pkgs/pkgs-lib/tests/default.nix

```diff
@@ -17,6 +17,7 @@ let
       jdk11 = pkgs.callPackage ../formats/java-properties/test { jdk = pkgs.jdk11_headless; };
       jdk17 = pkgs.callPackage ../formats/java-properties/test { jdk = pkgs.jdk17_headless; };
     };
+    libconfig = pkgs.callPackage ../formats/libconfig/tests.nix { };
```

Comment:
Apparently you can't `callPackage` something that isn't a package. If only the function was named in a manner to prevent people from calling it on non-packages :smile:

Reply:
It kinda works I think, but yeah it's not pretty.

Reply:
I think it's the reason behind the eval error. `callPackage` silently adds `override` and `overrideAttrs` to something which isn't an attrset, and it causes ofBorg to complain. I'll restructure it a bit.